ui/components/ui/box/box.js

```diff
@@ -358,4 +383,9 @@ Box.propTypes = {
    * Defaults to 'div'
    */
   as: PropTypes.string,
+  /**
+   * The color of the Typography component Should use the COLOR object from
+   * ./ui/helpers/constants/design-system.js
+   */
+  color: ValidTextColors,
```

Comment:
This should be taken care of in https://github.com/MetaMask/metamask-extension/pull/15507